components/x-gift-article/src/HighlightSection.jsx

```diff
@@ -33,14 +33,14 @@ export default ({
 	if (shareType === ShareType.enterprise && hasHighlights) {
 		if (isGiftUrlCreated && includeHighlights) {
 			return (
-				<div className="x-gift-article__checkbox">
+				<div className="x-gift-article__highlight_section">
```

Comment:
**nit-pick:** to continue the BEM pattern which you have used really well in your latest commit this should be `__highlight-section` rather than `__highlight_section`

Reply:
Thanks, I've updated the class name.  https://github.com/Financial-Times/x-dash/pull/708/commits/c95d3b0f2dc8b396466d4d5f163c6dfb8f44e93e